src/javascript/binary/pages/portfiliows.js

```diff
@@ -0,0 +1,181 @@
+var PortfolioWS =  (function() {
+
+    'use strict';
+
+    var pageLanguage = page.language();
+    if(!pageLanguage) pageLanguage = "EN";
+
+    var rowTemplate;
+    var retry;
+
+    var init = function() {
+        // get the row template and then discard the node as it has served its purpose
+        rowTemplate = $("#portfolio-dynamic tr:first")[0].outerHTML;
+        $("#portfolio-dynamic tr:first").remove();
+        TradeSocket.init({"thendo":"updateBalance"});
```

Comment:
I am not sure if it's a good idea to overload authorize api with other information, current implementation some what violates SRP, if later we want to remove the balance from authorize response, it will break your code

@raunak-binary @jy-binary 
any insights?